packages/core/src/components/Box/Box.types.ts

```diff
@@ -0,0 +1,22 @@
+export type Disabled = "opacityDisabled";
+
+export type Border = "border";
+
+export type BorderColor = "borderColorUiBorderColor" | "borderColorLayoutBorderColor";
```

Comment:
Any chance we can get rid of the `border` prefix here? Not a priory but if it's small maybe it's worth it

Reply:
Yes!